src/pages/index.js

```diff
@@ -15,69 +17,67 @@ import avatarSrc from "../images/jacques-cousteau.jpg";
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg ",
+import {
+  formSettings,
+  addButtonForm,
+  profileEditForm,
+  editAvatarForm,
+  profileTitle,
+  profileDescription,
+  profileTitleInput,
+  profileDescriptionInput,
+} from "../Utils/constants.js";
+
+// ********************************* API ******************************** //
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "16b910f9-e28d-483c-ac51-2ec35ebc8735",
+    "Content-Type": "application/json",
   },
-];
-
-// **************************** Elements **************************** //
-
-// Buttons
-const profileEditButton = document.querySelector("#profile-edit-button");
-const profileEditModal = document.querySelector("#profile-edit-modal");
-const addCardButton = document.querySelector("#add-card-button");
-
-// Profile
-const profileTitle = document.querySelector("#prof-title");
-const profileDescription = document.querySelector("#prof-description");
-const profileTitleInput = document.querySelector("#profile-title-input");
-const profileDescriptionInput = document.querySelector(
-  "#profile-description-input"
-);
-
-// Cards
-const addCardModal = document.querySelector("#add-card-modal");
+});
 
-// Modals
-const profileEditForm = profileEditModal.querySelector(".modal__form");
-const addButtonForm = addCardModal.querySelector(".modal__form");
+let userInfoInstance;
+
+// Initialiize, Generate User Profile and Cards on Load
+api
+  .getUserInfo()
+  .then((userInfo) => {
+    const { name, about, avatar } = userInfo;
+    userInfoInstance = new UserInfo(
+      profileTitle,
+      profileDescription,
+      avatarImg
+    );
+
+    userInfoInstance.setUserInfo({
+      name,
+      job: about,
+      avatar: avatar,
+    });
+    api.getInitialCards().then((res) => {
+      console.log(res);
+      res.forEach((item) => {
+        renderCard(item);
+      });
+    });
+  })
+  .catch((error) => {
+    console.log(error);
+  });
 
-// User Info Instantiation
-const userInfo = new UserInfo(profileTitle, profileDescription);
+// Edit User Profile
+const apiUpdateProfileInfo = (newInfo) => {
+  return api
+    .editUserInfo(newInfo)
+    .then((updatedInfo) => {
+      return updatedInfo;
+    })
+    .catch((error) => {
+      console.error(error);
+    });
```

Comment:
This `catch` is not needed here because you have it after the whole request chain in line 126